openshift/tektoncd-pipeline-operator
# [WIP] v0.9.2 canary release

Add make targets for:
    - test e2e
    - build operator image
    - push image
    - replace image in opeator deployment manifest
    - create new CSV (add to bundle)
    - operator-courier verify operator
    - push operator to quay application registry

Add v0.9.2 payload

Add v0.9.2 to canary channel

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>
